### PR TITLE
Display clearer error message when XAPI unreachable

### PR DESCRIPTION
**Pull request by benjamreis**
Differentiate when master is unreachable, XAPI service is down, no network configured and
also do not display an error when XAPI is finishing starting up

Signed-off-by: BenjiReis <benjamin.reis@vates.fr>

### CODE REVIEW

File: plugins-base/XSFeatureStatus.py [hunk 1]
```diff
@@ -15,9 +15,23 @@
 
 if __name__ == "__main__":
     raise Exception("This script is a plugin for xsconsole and cannot run independently")
-    
+
```
Comment (stormi) [inline plugins-base/XSFeatureStatus.py:18]: whitespace changes should belong in a separate PR

Comment (benjamreis) [reply plugins-base/XSFeatureStatus.py:18]: Right, I missed it.

File: plugins-base/XSFeatureStatus.py [hunk 2]
```diff
@@ -18,6 +18,20 @@
     
 from XSConsoleStandard import *
 
+def is_master():
+    master = False
+    try:
```
Comment (lindig) [inline plugins-base/XSFeatureStatus.py:23]: Python is not my main language but is this not a textbook case for `with open() as file`?

Comment (benjamreis) [reply plugins-base/XSFeatureStatus.py:22]: Yes, better code indeed.
I'm not sure about using a `try` around the `open` as well. if `pool.conf` can't be read, the exception should be raised no?

**Review by rosslagerwall — APPROVED** [review]

**Review by lindig — APPROVED** [review]
